src/components/composer/composer/composer.ts

```diff
@@ -364,6 +364,9 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
       const el = this.composerRef.el! as HTMLInputElement;
       content = el.childNodes.length ? el.textContent! : "";
     }
+    if (!content && this.props.focus === "inactive") {
```

Comment:
This probably works, but I think we should focus on the underlying cause of the issue: why the hell does the composer receive an input event when pressing enter on the link editor ?

Reply:
I think it's kind of the same beahaviour with the click event. [When you mousedown and and mouseup on differnet targets, the click event is ](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#:~:text=If%20the%20button%20is%20pressed%20on%20one%20element%20and%20the%20pointer%20is%20moved%20outside%20the%20element%20before%20the%20button%20is%20released%2C%20the%20event%20is%20fired%20on%20the%20most%20specific%20ancestor%20element%20that%20contained%20both%20elements.)
Worth looking into it at least

Reply:
After digging into it, I noticed that when we press enter in the link editor, it triggers the input event in the composer. According to the documentation, this doesn't quite match how the click event behaves.

I tried adding an onInput listener higher up in the hierarchy, like a spreadsheet component, and ran into the same problem. EventPhase tells us that it is happening in the target phase, not during bubbling or capturing.

I am unsure if this is the default behavior or not. Using PreventDefault stops it, but it's likely to cause other problems.

Reply:
I'd argue taht preventDefault is exactly what you are looking for. You want to stop the browser chain of events since it makes no sense to keep the "keyboard input event" behaviour once we closed (and destroyed) the input .